mmengine/optim/scheduler/param_scheduler.py

```diff
@@ -1283,3 +1283,203 @@ def get_position_from_periods(
             if iteration < period:
                 return i
         return None
+
+
+@PARAM_SCHEDULERS.register_module()
+class ReduceOnPlateauParamScheduler(_ParamScheduler):
+    """ReduceOnPlateauParamScheduler."""
+    _default_monitor_stages = ['val']
+
+    def __init__(
+            self,
+            optimizer: OptimizerType,
+            param_name: str,
+            monitor: str = 'val.loss',  # monitor_stage.monitor_key
```

Comment:
Why do we need to specify the `val.` prefix? Maybe we could not consider calling `ReduceOnPlateauParamScheduler.step` in `after_train_epcoh`.

Reply:
It is a bad design, I will modify it.